core/src/indexer/fetch_logs.rs

```diff
@@ -703,24 +728,51 @@ fn retry_with_block_range(
         });
     }
 
+    // Transient response errors, likely solved by halving the range or just retrying
+    if error_message.contains("response is too big")
+        || error_message.contains("error decoding response body")
+    {
+        let halved_to_block = halved_block_number(to_block, from_block);
+        return Some(RetryWithBlockRangeResult {
+            from: from_block,
+            to: halved_to_block,
+            max_block_range: None,
+        });
+    }
+
+    // We can't keep up with our own sending rate. This is rare, but we must backoff throughput.
+    if error_message.contains("error sending request") {
+        tokio::time::sleep(Duration::from_secs(3)).await;
```

Comment:
3 second a magic number?

Reply:
Yep, just a random backoff rate, dropped it to `1` for now.